### DISCUSSION TITLE
integration of fzf with fish for kubectl completion

### DISCUSSION THREAD

**Discussion by paulbhart** [junegunn/fzf#3897] [General]
I noticed some examples of how some have integrated fzf  and kubectl completion with bash or zsh [https://github.com/junegunn/fzf/wiki/examples#kubectl](https://github.com/junegunn/fzf/wiki/examples#kubectl), but have been unable to achieve the same thing for fish.  Anyone else able to figure this out, or have any other pointers?

**Comment by igrybkov** [comment]
Hey @paulbhart,

I don't know if you still need a solution, but I just spent a few hours figuring it out and documented it in the wiki. Feel free to use it:
```
type -q fzf >/dev/null 2>&1 && {
    kubectl completion fish | sed 's/-a \'\$__kubectl_comp_results\'$/-a \'(printf "%s\\\n" $__kubectl_comp_results | fzf --multi=0)\'/g' | source
}
```
